### DISCUSSION TITLE
How can I bump up PocketBase version?

### DISCUSSION THREAD

**Discussion by mtaku3** [pockethost/pockethost#179] [Q&A] [answered]
Hi, I think the version of PocketBase is not in the user's control.
How can I update the version?
Also, are there any restrictions on the versions that can be used?

**Comment by benallfree** [comment, marked as answer]
Hi, patch versions are bumped automatically. Minor auto bumps will likely happen after PocketBase reaches v1. 

If you want to go from 0.11 to 0.12 or something like that, just let me know and I can do it for you. 

You can also create a new instance and it will be at the latest. 

There are no version restrictions at this time.

**Reply by mtaku3** [reply]
Wonderful!
Can I ask for an update to 0.12.2 as mine is currently at 0.10.4?
My instance name is `automuteus-portable`.

**Reply by mtaku3** [reply]
@benallfree Hi! It has been a week since then, how is it going?

**Reply by benallfree** [reply]
@mtaku3 I'm so sorry, I was traveling. Your instance has been upgraded to `~0.12.0` (currently resolves to `0.12.3`).

**Reply by mtaku3** [reply]
Oh, I'm so sorry for mentioning you during your travel. And thanks a lot!

**Reply by NomadicDeveloper22** [reply]
@benallfree I seem to be stuck on 0.14.5 and having some OAuth issues that I think 0.15.1 might fix

instance: `ambitious-ghost`

**Reply by RA9** [reply]
@benallfree It will be great if you can update this `researchub` instance to the latest version of Pocketbase!

It will be a nice feature to update versions from the dashboard of pockethost with a click of a button

**Reply by benallfree** [reply]
@RA9 I'll handle this as quickly as I can, this is part of #60 and #72. If you want to take a look at either of those and submit a PR, you'll be helping a lot of other people too! 🎉
